### PR TITLE
Update version to 2.18.1

### PR DESCRIPTION
**Pull request by gnbm**
This PR is for updating version to 2.18.1

### Checklist

-   [X] tested locally
-   [ ] documented the code
-   [ ] clean all warnings and errors of eslint
-   [ ] requires changes in OutSystems (if so, provide a module with changes)
-   [ ] requires new sample page in OutSystems (if so, provide a module with changes)

### CODE REVIEW

File: dist/OutSystemsUI.css [hunk 1]
```diff
@@ -5264,14 +5236,12 @@ span.flatpickr-weekday{
...
   min-height:28px;
   overflow:auto;
-  -o-text-overflow:unset;
-     text-overflow:unset;
+  text-overflow:unset;
   white-space:normal;
 }
 .vscomp-wrapper.show-value-as-tags .vscomp-value-tag{
   overflow:hidden;
-  -o-text-overflow:ellipsis;
```
[the remark this answers is not in the document]
Comment (BenOsodrac) [reply dist/OutSystemsUI.css:5273]: Those browser prefixes are added automatically, probably in the meantime the Opera prefix for that is no longer needed, and in dev the builded theme didn't had that yet.

We don't even support Opera I believe

[the remark this answers is not in the document]
Comment (gnbm) [reply dist/OutSystemsUI.css:5273]: @BenOsodrac @rugoncalves  I just ran the build command and didn't change anything else. Why did this change?

**Review by joselrio — APPROVED** [review]

File: dist/OutSystemsUI.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (rugoncalves) [reply dist/OutSystemsUI.css:5273]: did we update the engine that compiles the scss files?

**Review by rugoncalves — APPROVED** [review]

File: dist/OutSystemsUI.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (BenOsodrac) [reply dist/OutSystemsUI.css:5273]: Not that I'm aware of. I think @joselrio has more context about this, he did the autoprefixer type feature on the repo. 

But I just tested to build on my side, and this rule isn't removed

[the remark this answers is not in the document]
Comment (BenOsodrac) [reply dist/OutSystemsUI.css:5273]: But it seems we use the latest version available btw: https://www.npmjs.com/package/gulp-autoprefixer

[the remark this answers is not in the document]
Comment (gnbm) [reply dist/OutSystemsUI.css:5273]: @joselrio any insight here before moving forward with the merge?

**Pull request merged**